src/components/selectionBoxLayer.ts

```diff
@@ -0,0 +1,116 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module Component {
+  export class SelectionBoxLayer extends AbstractComponent {
+    private _box: D3.Selection;
+    private _core: D3.Selection;
+    private _edgeT: D3.Selection;
+    private _edgeB: D3.Selection;
+    private _edgeL: D3.Selection;
+    private _edgeR: D3.Selection;
```

Comment:
No, since I need to address each of them individually. I could make a `Selection` or use an array and just assume the correct ordering, but that would make later code less clear as to what is actually happening.